### DISCUSSION TITLE
Problem in Jenkins installation

### DISCUSSION THREAD

**Discussion by VVSGanesh** [learndevopsonline/learndevopsonline#1550] [Q&A]
Hello Raghu,

This is Ganesh from b70 batch. I am facing issues after installing the jenkins I am getting the below error while trying to add the agent. No only while adding the agent this error I see in multiple places. I have used labauto to install jenkins and also used the process given in the jenkins documents still same. can you please help me further.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/96978469/c0175abb-5ce8-4488-aa45-5f0e4ee0a0cb)

**Comment by sreedharm07** [comment]
resolved in lab quary session
